libbeat/docs/dashboards.asciidoc

```diff
@@ -208,5 +208,5 @@ PS > .{backslash}{beatname_lc}.exe setup -e `
 ----
 endif::win_os[]
 
-endif::only-elasticsearch[]
+endif::no_logstash_output[]
```

Comment:
actually I did this intentionally because I thought it made sense to keep the endif text in situations where the conditional coding wasn't abundantly clear (for example, where there are nested conditions or a lot of text between the opening in closing tags). In other situations, I think the extra text actually makes the conditions harder to follow, so I removed it.